Library/Homebrew/test/utils/popen_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "utils/popen"
+
+describe Utils do
+  describe "::popen_read" do
+    it "reads the standard output of a given command" do
+      expect(subject.popen_read("sh", "-c", "echo success").chomp).to eq("success")
```

Comment:
This might just be because I'm on my phone (and therefore can't check exactly) but this line seems pretty long.